lib/graphs/poller-service-graph.js

```diff
@@ -116,6 +116,16 @@ module.exports = {
             }
         },
         {
+            label: 'ipmi-power-status-alert',
+            taskDefinition: {
+                friendlyName: 'IPMI Power Status Alerter',
+                injectableName: 'Task.Inline.Poller.Alert.Ipmi.PowerStatus',
+                implementsTask: 'Task.Base.Poller.Alert.Ipmi.PowerStatus',
```

Comment:
I haven't find this task in on-tasks repo. could you please help point it out?

Reply:
Sorry for the confusion. Two new created file was left behind while applying patch. These file are under on-tasks repo, and have been committed.
@iceiilin